pandas/tests/groupby/aggregate/test_other.py

```diff
@@ -421,11 +421,12 @@ def test_agg_timezone_round_trip():
     assert ts == grouped.nth(0)['B'].iloc[0]
     assert ts == grouped.head(1)['B'].iloc[0]
     assert ts == grouped.first()['B'].iloc[0]
-    assert ts == grouped.apply(lambda x: x.iloc[0])[0]
+    assert ts == grouped.apply(lambda x: x.iloc[0]).iloc[0, 0]
```

Comment:
can you reference this issue and maybe put a comment here on whey this is like this

Reply:
or this is actually returning a single column DataFrame right? can you assert that